### PR TITLE
mobile view back button

### PR DESCRIPTION
**Pull request by prachigarg19**
Add back button for width<=767px

Changes in this pull request:
- Add back button which when clicked will redirect user to callback_url parameter in the url.

### CODE REVIEW

File: src/Donations/index.tsx [hunk 1]
```diff
@@ -160,6 +166,24 @@ function DonationInfo() {
...
+            width: "fit-content",
+            position: "absolute",
+            zIndex: 1000,
+            background: "#fff",
+            padding: "7px",
+            borderRadius: "100%",
+          }}
+          onClick={goBack}
+        >
+          <BackButton color={"#000"} />
```
Comment (norbertschuler) [inline src/Donations/index.tsx:184]: There is a callback button for successful donations. I wonder which use case this pull requests solves. Can you describe what to do to reproduce this new code?

Comment (prachigarg19) [reply src/Donations/index.tsx:184]: The button you're referring to is displayed after the user has made a donation. 
This PR adds a back button in mobile view (<=767 px) which when clicked takes the user to callback_url parameter in the link  and is available in between the process of making a donation. 
To reproduce the result, try: https://donate-with-planet-rlvr6dqhs-planetapp.vercel.app/?callback_url=https%3A%2F%2Fplanet-webapp-2a398fy3i-planetapp.vercel.app%2Freforestemos-centro-sur

**Review by mariahosfeld — CHANGES_REQUESTED** [review]
Functionality generally works for me, but the following things need to be addressed:

- [ ] Styling of the back button, arrow is not centered and on my mobile phone it looks more like an oval, not a circle
grafik

- [ ] If the name of a project is too long, there is an overlap in the mobile view of the back button and the symbol above the name
grafik

**Comment by sunilsabatp** [comment]
donationBackbutton

File: src/Donations/index.tsx [hunk 2]
```diff
@@ -160,6 +166,26 @@ function DonationInfo() {
...
+            cursor: "pointer",
+            width: "fit-content",
+            zIndex: 1000,
+            background: "#fff",
+            padding: "7px",
+            borderRadius: "100%",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
```
Comment (sunilsabatp) [inline src/Donations/index.tsx:183]: pls do not use inline style instead define a class for it

**Comment by mariahosfeld** [comment]
Frontend functionality looks good to me.

**Review by sunilsabatp — APPROVED** [review]

**Review by mariahosfeld — APPROVED** [review]

**Pull request merged**